newrelic_agent.log

```diff
@@ -0,0 +1,8 @@
+{"v":0,"level":30,"name":"newrelic","hostname":"Adam-Levin-HP","pid":36612,"time":"2021-08-06T22:20:08.942Z","msg":"Using New Relic for Node.js. Agent version: 7.4.0; Node version: v14.16.1."}
```

Comment:
Doubt we want this file, will want to remove from the PR and add it to `.gitignore`

Reply:
File has been removed and added to ignore file.